src/shared/components/palette/history/event-log/event-log-table.component.spec.ts

```diff
@@ -112,8 +112,6 @@ describe('EventLogTableComponent', () => {
       const timeZoneOffset = - (new Date(date).getTimezoneOffset());
 
       expect(firstRowCells[COL_EVENT].nativeElement.textContent).toBe(firstEvent.event_name + firstEvent.significant_item.description);
-      expect(firstRowCells[COL_DATE].nativeElement.textContent)
```

Comment:
This was added to prevent a fail testing when not BST (at @andywilkinshmcts request). This could cause local unit testing to fail.